api-list.md

```diff
@@ -15,20 +34,19 @@
 ```
 API Root Path  
 |-- users
+|-- nodes (limited api)
 |-- organizations
 |   |-- nodes
 |   |-- occasions
 |   |-- containers
 |   |-- surveys
 |   |   |-- node
-|-- offerings
```

Comment:
I don't think we need a top level of offerings, do we? Maybe i'm wrong about that... i assume that's only for the "search by location" functionality?

Reply:
I made it top level simply because it would be accessed outside the context of an organization (i.e. by the public). I could see it as maybe being used for a say a plugin to Facebook or something? But I don't have a current use case that requires it.

Reply:
Ok, maybe for now let's leave it out. A good rule of thumb when creating an API is be very cautious adding new functionality because once it's added it's hard to take away